docs/developer/assets/css/custom.css

```diff
@@ -95,4 +95,11 @@ div.doc-contents:not(.first) {
   100% {
     background-position: 6rem 0; } }
 
+
+/* Fira code */
```

Comment:
Looks like we're in a TODO block, let's move it out

Reply:
Good catch, updated!